CHANGELOG.md

```diff
@@ -1,41 +1,61 @@
-CHANGELOG
-=========
-
-0.5.0 (2015-03-11)
-------------------
-
-* [BC] the package is now compatible for Laravel 5
-* improved the doc
-* add code of conduct
-
-0.4.1 (2014-06-23)
-------------------
-
-* fix the way to implode provider's arguments + unit tests
-
-0.4.0 (2014-04-13)
-------------------
-
-* use Geocoder 2.4.*
-
-
-0.3.0 (2014-04-13)
-------------------
-
-* support provider's arugments (BC break)
-
-
-0.2.0 (2013-11-16)
-------------------
-
-* use Geocoder 2.3.x
-* use config file
-* use singleton instead of share
-* improve tests
-
-
-0.1.0 (2013-09-16)
-------------------
-
-* add badges
-* initial import
+# Geocoder for Laravel Changelog
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+## [0.7.0] - 1 Oct 2016
+### Added
+- ability to dump results #16.
+- ability to use multiple providers in addition to the chain provider #47.
+- more integration tests.
+- special aggregator that allows chaining of `geocode()` and other methods.
+
+### Changed
+- README documentation.
+- to use Geocoder 3.3.x.
+- namespace to `Geocoder\Laravel\...`.
```

Comment:
Sounds good :) Should we start off with 1.0.0-RC as there are bound to be a few small wrinkles, like code coverage, additional test, and run it in a few projects?